apps/pie-docs/src/engineers/guidelines/browser-support.md

```diff
@@ -12,41 +12,41 @@ eleventyComputed:
 
 ## Browser support
 
-The number of browsers available to our users is vast, especially for those using mobile devices. Below, we have defined a rating system that helps us to specify the level of browser support we aim to achieve and to clarify what level of testing is required.
+Users have access to hundreds of browsers, so we've come up with a rating system to measure the level of browser support we want to achieve and the level of testing needed.
 
 {% notification {
 type: "warning",
-message: "As with any browser support checklist, this list should be reviewed often and updated in-line with the browsers and devices that our users are choosing to access our platforms."
+message: "This list needs to be reviewed often,  so we can stay up-to-date with the browsers our users are choosing."
```

Comment:
When used like this in a sentence it's not hyphenated, only if it's directly before a noun, i.e., "an up-to-date list" vs "keeping the list up to date".
```suggestion
message: "This list needs to be reviewed often to stay up to date with the browsers currently chosen by our users."
```